tests/crowdsourcing/tasks/model_chat/test_model_chat.py

```diff
@@ -132,20 +133,8 @@ def test_base_task(self):
                     agent_messages=AGENT_MESSAGES,
                     form_messages=FORM_MESSAGES,
                     form_task_data=FORM_TASK_DATA,
-                    expected_states=(expected_state,),
                     agent_task_data=AGENT_TASK_DATA,
-                )
-
-                # Check that the contents of the chat data file are as expected
-                with open(expected_chat_data_path) as f:
-                    expected_chat_data = json.load(f)
-                results_path = list(
-                    glob.glob(os.path.join(chat_data_folder, '*/*_*_*_sandbox.json'))
-                )[0]
-                with open(results_path) as f:
-                    actual_chat_data = json.load(f)
-                self._check_final_chat_data(
-                    actual_value=actual_chat_data, expected_value=expected_chat_data
+                    data_regression=data_regression,
```

Comment:
It looks like `expected_state` on line 129 is never used, because of course we're using data_regression here - I'd get rid of that